### DISCUSSION TITLE
CF API changes

### DISCUSSION THREAD

**Discussion by layday** [ogri-la/wowman#313] [General]
Just a heads-up that CF plan to:

1. have download URLs from the API not count towards author credits
1. give authors the option to unlist their add-ons from the API
1. rate limit the API
1. require authentication - not for end-users but for add-on managers
1. issue API keys following a review process where they ask for your real name and motivations
1. require that keys do not leak

Under the current plan the new API will be rolled out in Dec and the old API endpoint retired in Feb.

**Comment by layday** [comment]
The announcement: https://mailchi.mp/overwolf/whats-new-with-overwolf-curseforge-november3

**Comment by torkus** [comment]
Ah - this might explain a few weird upticks in stars and downloads in the last few days - as well as your list of github addons.

I'll give it a read but I think we had a gut feeling this was coming.
